odoo/addons/base/models/ir_actions.py

```diff
@@ -389,6 +411,7 @@ class IrActionsServer(models.Model):
 #  - time, datetime, dateutil, timezone: useful Python libraries
 #  - float_compare: Odoo function to compare floats based on specific precisions
 #  - log: log(message, level='info'): logging function to record debug information in ir.logging table
+#  - _logger: _logger.info(message): logging class to record debug information in Odoo's log
```

Comment:
Are we sure we want to document this one?

Reply:
> Are we sure we want to document this one?

> "oui a priori si c'est visible dans le code, autant que ce soit documenté" (ODO)

See: https://discord.com/channels/678381219515465750/687339689522364423/877929059009577021